### PR TITLE
Fedora31 integration test fixes

### PR DESCRIPTION
**Pull request by janpriessnitz**
- python related fixes for integration tests
- 089 external cache manager test fix

### CODE REVIEW

**Comment by janpriessnitz** [comment]
@cernvm-bot cloudtest nodestroy

**Comment by janpriessnitz** [comment]
@cernvm-bot cloudtest

**Comment by janpriessnitz** [comment]
@cernvm-bot cloudtest nodestroy

**Comment by janpriessnitz** [comment]
@cernvm-bot cloudtest

**Comment by janpriessnitz** [comment]
@cernvm-bot unittest

**Comment by janpriessnitz** [comment]
@cernvm-bot cloudtest

**Comment by janpriessnitz** [comment]
@cernvm-bot cloudtest

File: test/test_functions [hunk 1]
```diff
@@ -1029,7 +1033,7 @@ open_http_server() {
   local port="$2"
   local logfile="${3:=/dev/null}"
 
-  local http_server="${TEST_ROOT}/common/mock_services/http_server.py"
+  local http_server="python2 ${TEST_ROOT}/common/mock_services/http_server.py"
```
Comment (jblomer) [inline test/test_functions]: Is this still going to change or is the PR ready to merge?

**Comment by janpriessnitz** [comment]
I'm still looking into the macOS issue. I'll delete the last commit so you can merge it ASAP. In any case, I need to review the python paths further since there are more python-dependent tests not in the quick suite.

**Comment by janpriessnitz** [comment]
@jblomer ready to merge

**Pull request merged**